RobotCode/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -1,17 +1,31 @@
 package frc.robot;
 
+import edu.wpi.first.wpilibj.Joystick;
 import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
 import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
 import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
 import edu.wpi.first.wpilibj2.command.Command;
 
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+import frc.robot.commands.RotateShooterMountCommand;
+import frc.robot.subsystems.ShooterMountSubsystem;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+
 public class RobotContainer
 {
 
 	private final static ShuffleboardTab shuffleboard = Shuffleboard.getTab("Tab 1");
 
 	private final SendableChooser<Command> autoChooser = new SendableChooser<>();
 
+	private Joystick secondaryController;
```

Comment:
Not used outside of `configureSecondaryBindings`, so just make it a variable in there, not as part of the class.